Calibration/HcalAlCaRecoProducers/python/ALCARECOHcalCalMinBiasHI_cff.py

```diff
@@ -6,56 +6,132 @@
 
 import EventFilter.HcalRawToDigi.HcalRawToDigi_cfi
 hcalDigiAlCaMB = EventFilter.HcalRawToDigi.HcalRawToDigi_cfi.hcalDigis.clone()
-import RecoLocalCalo.HcalRecProducers.HcalSimpleReconstructor_hbhe_cfi
-hbherecoNoise = RecoLocalCalo.HcalRecProducers.HcalSimpleReconstructor_hbhe_cfi.hbheprereco.clone()
-import RecoLocalCalo.HcalRecProducers.HcalSimpleReconstructor_hf_cfi
-hfrecoNoise = RecoLocalCalo.HcalRecProducers.HcalSimpleReconstructor_hf_cfi.hfreco.clone()
-import RecoLocalCalo.HcalRecProducers.HcalSimpleReconstructor_hf_cfi
-hfrecoMBspecial = RecoLocalCalo.HcalRecProducers.HcalSimpleReconstructor_hf_cfi.hfreco.clone()
-import RecoLocalCalo.HcalRecProducers.HcalSimpleReconstructor_ho_cfi
-horecoNoise = RecoLocalCalo.HcalRecProducers.HcalSimpleReconstructor_ho_cfi.horeco.clone()
+
+hcalDigiAlCaMB.firstSample = 0
+hcalDigiAlCaMB.lastSample = 9
+hcalDigiAlCaMB.InputLabel = 'rawDataCollector'
+
 #add GT digi:
 import EventFilter.L1GlobalTriggerRawToDigi.l1GtUnpack_cfi
 gtDigisAlCaMB = EventFilter.L1GlobalTriggerRawToDigi.l1GtUnpack_cfi.l1GtUnpack.clone()
 
+gtDigisAlCaMB.DaqGtInputTag = 'source'
+
 import HLTrigger.HLTfilters.hltHighLevel_cfi
 hcalminbiasHLT =  HLTrigger.HLTfilters.hltHighLevel_cfi.hltHighLevel.clone(
 #    HLTPaths = ['HLT_HcalPhiSym'],
     eventSetupPathsKey='HcalCalMinBiasHI',
     throw = False #dont throw except on unknown path name 
 )
 
-seqALCARECOHcalCalMinBias = cms.Sequence(hcalminbiasHLT*hcalDigiAlCaMB*gtDigisAlCaMB*hbherecoNoise*hfrecoNoise*hfrecoMBspecial*horecoNoise)
+import RecoLocalCalo.HcalRecProducers.HBHEPhase1Reconstructor_cfi
+hbherecoNoise = RecoLocalCalo.HcalRecProducers.HBHEPhase1Reconstructor_cfi.hbheprereco.clone(
+    digiLabelQIE8  = cms.InputTag("hcalDigiAlCaMB"),
+    digiLabelQIE11 = cms.InputTag("hcalDigiAlCaMB"),
+    tsFromDB = cms.bool(False),
+    dropZSmarkedPassed = cms.bool(False),
+    algorithm = dict(
+        useM2 = cms.bool(False),
+        useM3 = cms.bool(False)
+    ),
+    processQIE11 = cms.bool(False),
+    setNegativeFlagsQIE8 = cms.bool(False),
+    setNegativeFlagsQIE11 = cms.bool(False),
+    setNoiseFlagsQIE8 = cms.bool(True),
```

Comment:
This came after a disussion between Igor (who has rewritten the reconstruction code) and Olga/myself. We only would permit negative values of ADC channels but allow noise rejection